### PR TITLE
upgrade to recent chainweb (including 20 chain graph)

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+package aeson
+    flags: +cffi
+
+debug-info: True
+
 source-repository-package
     type: git
     location: https://github.com/kadena-io/chainweb-node.git
-    tag: c9d8c1112ef0093bca04f6124721d8fa41ebf5a9
+    tag: 3c5f2264520a3edccf4731436b3a0545098d85a9
```
Comment (mightybyte) [inline cabal.project]: Should we bump this to the 2.0 commit?

File: cabal.project [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 
 source-repository-package
     type: git
     location: https://github.com/kadena-io/pact.git
-    tag: ace18a2489bf86595af09316bd1c09794c326b77
+    tag: 2ca4cae3eb8a0ef2783aa9457388c31b44a0dfbc
```
Comment (mightybyte) [inline cabal.project:16]: Same here for whatever version of pact chainweb 2.0 is using

File: stack.yaml [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
+  - sbv-8.6
   - github: kadena-io/thyme
     commit: 6ee9fcb026ebdb49b810802a981d166680d867c9
 
   # --- Custom Pins --- #
   - github: kadena-io/chainweb-node
-    commit: ce4f88a254caab708857123e8294d795ad335d45
-  - github: kadena-io/chainweb-storage
-    commit: 17a5fb130926582eff081eeb1b94cb6c7097c67a
+    commit: 3c5f2264520a3edccf4731436b3a0545098d85a9
```
Comment (mightybyte) [inline stack.yaml]: Same comment here as above.

**Review by mightybyte — COMMENTED** [review]
Made a couple comments about git commits.  We should also update the kpkgs pin to match the other deps.

File: cabal.project (hunk 2 above)
Comment (larskuhtz) [reply cabal.project:16]: That one is already up to date

**Pull request merged**